Games/types/Mafia/roles/cards/MakeGuilty.js

```diff
@@ -0,0 +1,24 @@
+const Card = require("../../Card");
+const { PRIORITY_MAKE_GUILTY } = require("../../const/Priority");
+
+module.exports = class MakeInnocent extends Card {
```

Comment:
Change the card name here

Reply:
This however, was not deliberate :P
Nice catch